extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/test/java/io/quarkus/resteasy/reactive/server/test/multipart/MultipartOutputResource.java

```diff
@@ -11,7 +11,7 @@
 import javax.ws.rs.core.MediaType;
 
 import org.jboss.resteasy.reactive.RestResponse;
-import org.jboss.resteasy.reactive.server.core.multipart.MultipartFormDataOutput;
+import org.jboss.resteasy.reactive.server.multipart.MultipartFormDataOutput;
```

Comment:
Would it be hard to keep the old class for a few versions as deprecated and then drop it? (If we can easily extend the new one and have things working.)

Reply:
I tried initially, but it turned out to be a big hassle so I opted against it (especially considering it's just a package change)

Reply:
OK, if it's too much of a hassle, I totally agree. Let's make sure we drop a line there: https://github.com/quarkusio/quarkus/wiki/Migration-Guide-2.17 when merged.

Reply:
Absolutely